typings/blockly.d.ts

```diff
@@ -1061,6 +1061,7 @@ declare namespace Blockly {
         let flyoutCategory: (wp: Workspace) => HTMLElement[];
         let flyoutCategoryBlocks: (wp: Workspace) => HTMLElement[];
         function createVariable(wp: Workspace, opt_callback?: ((e: any) => void)): void;
+        function allUsedVarModels(ws: Blockly.Workspace): Array<Blockly.VariableModel>;
```

Comment:
For arrays, we typically use the syntax:
```typescript
Blockly.VariableModel[]
```
instead of the generic angle-brackets

Reply:
Cool, changed.